docusaurus/docs/iOS/05-ui-cookbook/15-long-press-to-focus.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Long Press to Focus
+description: Documentation on implementing a long press gesture to focus on a particular user in a StreamVideo call.
+---
+
+# Long Press to Focus
+
+The StreamVideo SDK allows for interactive and intuitive user engagement during calls. Implementing a long press gesture to focus your local participant's video feed can significantly enhance the user experience.
+
+## Overview
+
+- **Gesture Recognition**: Detect a long press on a participant's video feed.
+- **Focus Implementation**: Focus on the specified point on your video feed.
+- **User Experience**: Intuitive interaction for a more immersive call experience.
+
+## Implementing Long Press to Focus
+
+### Detecting the Long Press Gesture
+
+You can find an implementation that we are using in our Demo app that focus on longPress. To achieve that we are using the following ViewModifier
```

Comment:
- maybe "to focus on long press", or "that focuses on long press"?
- `ViewModifier`
-  "." or ":" missing at the end.